### ISSUE TITLE
Yaml.stringify()

### ISSUE THREAD

**Issue by dgzlopes** [awslabs/cdk8s#419]
### Description of the feature or enhancement:
I would love to have some YAML utilities built-in to cdk8s.

### Use Case:

While working with k8s, we are working with YAML in one way or another. Having some YAML utilities would enable users to do things like this one, without having to install external deps:
```typescript
    const datasources = new ConfigMap(this, 'grafana-datasources', {
      data: {
        "loki.yaml": yaml.safeDump({ 
          apiVersion: 1, 
          datasources: [{
            name: 'Loki', 
            type: 'loki', 
            access: 'proxy', 
            url: `${loki.releaseName}:3100`, 
            jsonData: { maxLines: 1000 } 
          }]
        })
      }
    });
```

### Other:

This was proposed by @eladb on Slack.

* [ ] :wave: I may be able to implement this feature request -> I don't have enough bandwidth right now :(
* [ ] :warning: This feature might incur a breaking change

---

This is a :rocket: Feature Request

**Comment by eladb** [comment]
There is a `Yaml` class with a bunch of utilities: https://cdk8s.io/docs/v1.0.0-beta.5/reference/cdk8s/#cdk8s-yaml

**Comment by dgzlopes** [comment]
Thanks for taking a look @eladb! 

I've read the docs of the Yaml class, and I see that we don't have anything to convert Javascript data structures to a YAML string (as `yaml.safeDump` does). I illustrated my use-case above, on the first comment!

**Comment by eladb** [comment]
@dgzlopes Yes you are correct. We don't expose this method. Should be trivial. 

Contributions are welcome.
